src/Compilers/VisualBasic/Portable/Binding/SourceFileBinder.vb

```diff
@@ -81,6 +81,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Public Overrides ReadOnly Property CheckOverflow As Boolean
+            Get
+                ' VBCore should always be compiled with overflow checks on.
+                If SyntaxTree.GetEmbeddedKind() = EmbeddedSymbolKind.VbCore Then
```

Comment:
Sure, I'll do that. In VB overflow checks are off by default, so I am not sure whether this optimization will be advantageous in a common case, it might actually have an opposite impact.